StackExchange.Redis/StackExchange/Redis/ConnectionMultiplexer.cs

```diff
@@ -37,13 +37,16 @@ public static Task<T> ObserveErrors<T>(this Task<T> task)
             return task;
         }
 
+        [ThreadStatic]
+        public static bool ContinueOnCapturedContext;
```

Comment:
In this case I am providing an alternative threading model. The await callbacks are queued to a local queue that the thread calling connect services until completion.